build/scripts/generate_latest_metas.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+#
+# Copyright (c) 2018-2019 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Utility for automatically generating latest meta.yamls for plugins.
+#
+
+set -e
+
+for plugin_dir in v3/plugins/*/*; do
+  echo "Updating plugin ${plugin_dir}"
+  # Make sure every plugin has a 'latest.version' file
+  if [ ! -d "${plugin_dir}"/latest ] || [ ! -f "${plugin_dir}"/latest/latest.version ]; then
+    echo "    Missing latest.version: ${plugin_dir}/latest/latest.version"
```

Comment:
I might prefer if we use `latest.txt` or `latest/version.txt` instead of `latest/latest.version` but :man_shrugging: doesn't change the functionality here.

Reply:
Done.